config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   get '/about', to: 'about#show'
   get '/info', to: 'info#show'
+
+  get '/login', to: 'sessions#new'
+  get '/auth/:provider/callback', to: 'sessions#create'
+  get '/auth/google', as: 'google_login'
+  delete '/logout', to: 'sessions#destroy' 
```

Comment:
should we group the login/logout just to keep it organized/clean??

Reply:
I agree, it does look messy.  I'll address it the next time I'm in the routes file.